crates/node-file-trace/src/lib.rs

```diff
@@ -259,6 +259,7 @@ async fn input_to_modules<'a>(
         env,
         ModuleOptionsContext {
             enable_types: true,
+            enable_mdx: false,
```

Comment:
false is the default, so this is not really needed. But you need to set it to `true` for next-dev ModuleOptionContextes